src/client/Microsoft.Identity.Client.Broker/RuntimeBroker.cs

```diff
@@ -94,6 +109,60 @@ private static void OnProcessExit(object sender, EventArgs e)
             }
         }
 
+        private void LogEventRaised(NativeInterop.Core sender, LogEventArgs args)
+        {
+            LogLevel msalLogLevel = _logLevelMap[args.LogLevel];
+            if (_logger.IsLoggingEnabled(msalLogLevel))
+            {
+                if (_logger.PiiLoggingEnabled)
+                {
+                    switch (msalLogLevel)
+                    {
+                        case LogLevel.Always:
```

Comment:
nit: you can get rid of the switches and just use 

`logger.Log(LogLevel, messageWithPii, messageWithoutPii)`